src/test/java/com/missafternoon/budjen/transactions/JSONInterpreterTest.java

```diff
@@ -0,0 +1,102 @@
+package com.missafternoon.budjen.transactions;
+
+import com.missafternoon.budjen.formatting.DateConverter;
+import org.json.JSONException;
+import org.junit.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class JSONInterpreterTest {
+
+    private JSONInterpreter target = new JSONInterpreter();
+    private TestTransactionData testData = new TestTransactionData();
+
+    @Test
+
+    public void writeJavaObjectIntoJSONString() throws JSONException {
+        String json = target.convertToString(testData.getTransactionData());
+        String expectedJson = "[" +
+                "{\"type\":\"DEBIT\",\"description\":\"Plastic Surgery\",\"money\":-2500000,\"date\":\"2018-06-13\"}," +
+                "{\"type\":\"CREDIT\",\"description\":\"Let flat\",\"money\":420000,\"date\":\"2018-06-08\"}" +
+                "]";
+        JSONAssert.assertEquals(expectedJson, json, JSONCompareMode.STRICT);
```

Comment:
Cool library. I'm actually working on writing a similar library in Go (with the same name :( ). Is this a transitive dependency of jackson? (I.e. Jackson requires it, and we require jackson, thus we require it). 

Not sure strict is what we want here - it's too brittle w.r.t. the ordering of the keys in the JSON objects. I assume you're using it to ensure that the order of the JSON array is preserved - in which case I think we're better off extracting each element based on index and asserting objects against objects without the STRICT mode.